File: Data/scripts/stopwatch.cs
using ImGuiNET;
using System.Diagnostics;

class Script
{
    Stopwatch Stopwatch = new Stopwatch();

    public void Draw()
    {
        if (ImGui.Begin("Stopwatch"))
        {
            var ts = Stopwatch.Elapsed;

            string elapsedTime = string.Format("{0:00}:{1:00}:{2:00}.{3:00}",
                ts.Hours, ts.Minutes, ts.Seconds,
                ts.Milliseconds / 10);

            ImGui.Text(elapsedTime);

            if (!Stopwatch.IsRunning)
            {
                if (ImGui.Button("Start"))
                {
                    Stopwatch.Start();
                }
            }
            else
            {
                if (ImGui.Button("Pause"))
                {
                    Stopwatch.Stop();
                }
            }
            ImGui.SameLine();
            if (ImGui.Button("Reset"))
            {
                Stopwatch.Reset();
            }

        }
        ImGui.End();
    }
}

File: Data/scripts/stopwatch.js
let stopwatch = new System.Diagnostics.Stopwatch();

function onDraw()
{
    if (ImGui.Begin("Stopwatch"))
    {
        let ts = stopwatch.Elapsed;

        let elapsedTime = System.String.Format(
            "{0:00}:{1:00}:{2:00}.{3:00}",
            ts.Hours, ts.Minutes, ts.Seconds,
            ts.Milliseconds / 10
        );

        ImGui.Text(elapsedTime);

        if (!stopwatch.IsRunning)
        {
            if (ImGui.Button("Start"))
                stopwatch.Start();
        }
        else
        {
            if (ImGui.Button("Pause"))
                stopwatch.Stop();
        }

        ImGui.SameLine();

        if (ImGui.Button("Reset"))
            stopwatch.Reset();
    }
    ImGui.End();
}